js/bittrex.js

```diff
@@ -504,6 +527,120 @@ module.exports = class bittrex extends Exchange {
         });
     }
 
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let market = undefined;
+        await this.loadMarkets ();
+        let request = {};
+        if (symbol !== undefined)
+            request['symbol'] = this.market (symbol)['id'];
+        if (limit !== undefined)
+            request['count'] = limit;
+        let response = await this.accountGetOrderhistory (this.extend (request, params));
```

Comment:
The same is already done in fetchClosedOrders for Bittrex, so, I guess, they don't have an endpoint for fetchMyTrades? This was the reasoning behind setting has['fetchMyTrades'] to false. We actually had it implemented in the same way previously, but, in reality, orders don't always match one-to-one, therefore each order contains one or more trades. Of course, Bittrex hides it from the user with their API, and, ultimately, the users sees those fills as one-to-one. But we think it's better to have as less code in the lib as possible, that is, if the users wants to derive trades from orders, then it is probably better to do that in userland? I'm interested in what you think about it )

Reply:
I was not aware that took place.

So after doing some digging into data, apis, and talking it over with a colleague. We realized that yes they do not have a fetch my trades endpoint. In fact we could find no way to tie individual trades with orders, in any retrospective way. Also the bittrex API is quite terrible in a lot of ways in getting useful detailed data efficiently. 

With that in mind I am going to rip out the `fetchMyTrades` work, it just isn't a good fit. Maybe with the new API they will add it, but after digging through the UI and csv exports. I doubt they will.

Reply:
Yeah, no way to paginate through our own history with Bittrex is a bit dissapointing... Not mentioning the other issues with it... But I hope to see at least some improvements from them by the end of the year )